src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1564,7 +1564,7 @@ public function getContent($asResource = false)
      */
     public function getETags()
     {
-        return preg_split('/\s*,\s*/', $this->headers->get('if_none_match'), null, \PREG_SPLIT_NO_EMPTY);
+        return preg_split('/\s*,\s*/', $this->headers->get('if_none_match'), -1, \PREG_SPLIT_NO_EMPTY);
```

Comment:
[preg_split](https://www.php.net/manual/en/function.preg-split.php)'s third argument supports `null` in PHP 7. However, in PHP 8 it is required to be an int. 

From PHPStorm's stubs: 

> If specified, then only substrings up to <i>limit</i> are returned with the rest of the string being placed in the last substring. A <i>limit</i> of -1, 0 or <b>NULL</b> means "no limit" and, as is standard across PHP, you can use <b>NULL</b> to skip to the <i>flags</i> parameter.

From the docs (PHP 8 ready): 

> If specified, then only substrings up to limit are returned with the rest of the string being placed in the last substring. A limit of -1 or 0 means "no limit".